### PR TITLE
Add get_attname_column method of models.Field

### PR DESCRIPTION
**Pull request by gg-mmill**
Add type declaration of the get_attname_column of models.Field

See source in django: https://github.com/django/django/blob/main/django/db/models/fields/__init__.py#L973

### CODE REVIEW

**Review by sobolevn — COMMENTED** [review]
Please, adjust https://github.com/typeddjango/django-stubs/blob/master/scripts/stubtest/allowlist_todo.txt

File: scripts/stubtest/allowlist_todo.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# Autogenerated by `stubtest`
```
Comment (sobolevn) [inline scripts/stubtest/allowlist_todo.txt]: Please, keep this line :)

File: scripts/stubtest/allowlist_todo.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
...
 # Unsorted: there are real problems and things we can really ignore.
```
Comment (sobolevn) [inline scripts/stubtest/allowlist_todo.txt]:
```suggestion
# Autogenerated by `stubtest`
# Unsorted: there are real problems and things we can really ignore.
```

File: scripts/stubtest/allowlist_todo.txt (hunk 1 above)
Comment (gg-mmill) [reply scripts/stubtest/allowlist_todo.txt]: Indeed, wasn't on purpose :/

**Review by sobolevn — APPROVED** [review]
Thank you! 👍

**Pull request merged**